cmd/afsPmf/afsPmf.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/popgen"
+	"log"
+)
+
+func afsPmf(vcfFile string, outFile string, unPolarized bool) {
+	g, err := popgen.VcfToAfs(vcfFile, !unPolarized) //VcfToAfs is written in terms of polarized, so this is inverted here.
+	exception.FatalOnErr(err)
+	out := fileio.EasyCreate(outFile)
+	defer out.Close()
```

Comment:
should really be handling this error. EasyWriters have some pretty serious close errors (e.g. could not write all of buffer)